src/pages/workspace/WorkSpaceUtils.js

```diff
@@ -0,0 +1,15 @@
+import lodashGet from 'lodash/get';
+import BankAccount from '../../libs/models/BankAccount';
+import Navigation from '../../libs/Navigation/Navigation';
+
+function getShouldShowPage(props, nextPage, defaultPage) {
+    const state = lodashGet(props.reimbursementAccount, 'achData.state');
+    const isShowPage = lodashGet(props.reimbursementAccount, 'achData.bankAccountID') && state !== +BankAccount.STATE.OPEN;
```

Comment:
```suggestion
    const shouldShowPage = lodashGet(props.reimbursementAccount, 'achData.bankAccountID') && state !== +BankAccount.STATE.OPEN;
```

Why is this + in `+BankAccount.STATE.OPEN`?

Reply:
Removed.

Reply:
I don't see this suggestion applied.